### PR TITLE
Fixes navigation computers giving too many port jumping actions

### PR DESCRIPTION
**Pull request by LemonInTheDark**
## About The Pull Request

We never clear the actions list, so this was real nasty 

I'm adding a vestigial proc here, remove_jumpable_port
I prefer this over not fully implementing the behavior, hopefully it'll be of use to someone

### CODE REVIEW

File: code/modules/shuttle/navigation_computer.dm [hunk 1]
```diff
@@ -57,6 +60,25 @@
 	else
 		QDEL_NULL(my_port)
 
+/// "Initializes" any default port ids we have, done so add_jumpable_port can be a proper setter
+/obj/machinery/computer/camera_advanced/shuttle_docker/proc/set_init_ports()
+	var/list/init_ports = jump_to_ports.Copy()
+	jump_to_ports = list() //Reset it so we don't get dupes
+	for(var/port_id in init_ports)
+		add_jumpable_port(port_id)
```
Comment (MrMelbert) [inline code/modules/shuttle/navigation_computer.dm:68]: In the PR description  you said you added 1 vestigial proc, `remove_jumpable_port` which I got

But I don't see this being called anywhere, is it?

File: code/modules/shuttle/navigation_computer.dm [hunk 2]
```diff
@@ -57,6 +60,25 @@
...
+		
+/obj/machinery/computer/camera_advanced/shuttle_docker/proc/add_jumpable_port(port_id)
+	if(!length(jump_to_ports))
+		actions += new /datum/action/innate/camera_jump/shuttle_docker(src)
+	jump_to_ports[port_id] = TRUE
```
Comment (MrMelbert) [inline code/modules/shuttle/navigation_computer.dm:73]: How does this action ever get added for mapload shuttle consoles?

Seems like `add_jumpable_port` is only called in `Initialize` from within a `if(!mapload)` conditional

File: code/modules/shuttle/navigation_computer.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/modules/shuttle/navigation_computer.dm:68]: I uh. I was going to call it in Initialize. but I forgot to use it

File: code/modules/shuttle/navigation_computer.dm (hunk 2 above)
Comment (LemonInTheDark) [reply code/modules/shuttle/navigation_computer.dm:73]: See above

**Pull request merged**